client/src/controllers/DropdownController.ts

```diff
@@ -104,4 +165,12 @@ export class DropdownController extends Controller<HTMLElement> {
       },
     };
   }
+
+  get plugins() {
+    return [
+      hideTooltipOnBreadcrumbsChange,
+      hideTooltipOnEsc,
+      rotateToggleIcon,
+    ].concat(this.hideOnClickValue ? [hideTooltipOnClickInside] : []);
```

Comment:
Hahaha, the lengths we go to for JS expressions over statements!

Reply:
Yeah. True, it's probably a bit esoteric to write this way.